client/router.js

```diff
@@ -344,7 +344,10 @@ Router.prototype._notfoundRoute = function(context) {
   // XXX this.notfound kept for backwards compatibility
   this.notFound = this.notFound || this.notfound;
   if(!this.notFound) {
-    console.error("There is no route for the path:", context.path);
+    if(!Meteor.isTest && !Meteor.isPackageTest) {
+      console.error("There is no route for the path:", context.path);
+    }
```

Comment:
Add a else branchement with a log maybe ?

    } else {
    	console.log("There is no route for the path:", context.path);
    	if (window.console.trace) {
    		console.trace();
    	}
    }

Otherwise :+1: